### PR TITLE
 Add support for XMR 

### PR DESCRIPTION
**Pull request by aranhaagency**
close https://github.com/ensdomains/ens-app/issues/912

Monero Base58 Docs https://monerodocs.org/cryptography/base58/
Monero Address Generate:  https://xmr.llcoins.net/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ This library currently supports the following cryptocurrencies and address forma
  - BCH (base58check and cashAddr; decodes to cashAddr)
  - BNB (bech32)
  - XLM (ed25519 public key)
+ - XMR (base58xmr public key)
```
Comment (makoto) [inline README.md:44]: Can you provide reference link so that we can verify?

Comment (aranhaagency) [reply README.md:44]: @makoto 
Monero Base58 Docs https://monerodocs.org/cryptography/base58/
Monero Address Generate:  https://xmr.llcoins.net/

**Comment by Arachnid** [comment]
This PR grows the library from 170k to 214k - an increase of 25% for one coin. Are you able to restructure it so a single base58 library that supports both encoding variants is used, or pull only the necessary code into this library (appropriately attributed and copyright-noticed)?

**Comment by brantlymillegan** [comment]
@aranhaagency Hey just checking in. Are you still interested in completing this PR/bounty?

**Comment by brantlymillegan** [comment]
Due to inactivity I'm going to close and relist this bounty. @aranhaagency Feel free to resubmit with the requested changes with the new bounty

**Pull request closed without merging**